src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/FileExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.FileStoragePathUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions
+{
+    public static class FileExtensions
+    {
+        public static string Path(this ReleaseFile releaseFile)
+        {
+            return releaseFile.ReleaseFileReference.Path();
+        }
+
+        public static string Path(this ReleaseFileReference file)
+        {
+            return AdminReleasePath(
+                file.ReleaseId,
+                file.ReleaseFileType,
+                file.BlobStorageName);
+        }
+
+        public static string PublicPath(this ReleaseFileReference file, string publicationSlug, string releaseSlug)
```

Comment:
Shouldn't we use the `file.Release.Publication` relationship to get these slugs? Again, conscious that this method could be misused to create the wrong path.

Reply:
As above, `ReleaseFileReference.Release` is the `Release` that the file was originally uploaded against and doesn't give us the slugs to correctly locate files in public storage.

Reply:
It would be nice if we can get rid of publication slug and release slug altogether in a future migration.